netlify.toml

```diff
@@ -0,0 +1,10 @@
+# Global settings applied to the whole site.
+[build]
+  base    = "/"
+  publish = "app/"
+  command = "npm install && npm run setup"
+
+# Deploy Preview context: All Deploy Previews
+# will inherit these settings.
+[context.set-api.environment]
+  APIHOST = "https://104.199.16.253/api/"
```

Comment:
just to avoid the scenario in which we merge back to master the wrong API url, should we use their `branch contexts` ? Something along the lines of 
```
# Production context: All deploys to the main
# repository branch (master in our case) will inherit these settings.
[context.production.environment]
    APIHOST = "https://api.opentargets.io/"

# Deploy Preview context: All Deploy Previews
# will inherit these settings. (All PRs)
[context.deploy-preview.environment]
  APIHOST = "https://35.187.191.205/api/"

# Branch Deploy context: All deploys that are not in
# an active Deploy Preview will inherit these settings.
[context.branch-deploy.environment]
  APIHOST = "https://35.187.191.205/api/"
```
You still have to change your `APIHOST` var, but the master one is left untouched.